app/src/main/java/io/gnosis/safe/ui/transaction/TransactionsViewModel.kt

```diff
@@ -19,7 +19,8 @@ class TransactionsViewModel
     fun load() {
         safeLaunch {
             val safeAddress = safeRepository.getActiveSafe()!!.address
-            val transactions = transactionRepository.getTransactions(safeAddress)
+            val safeInfo = safeRepository.getSafeInfo(safeAddress)!!
```

Comment:
safeRepository should give back the non-nullable object. Every added safe on the device should be valid and have safe info.